mattermost/mattermost-webapp
# MM-16232 Add E2E to verify system config for Push Notification Contents

#### Summary
Add E2E to verify system config for Push Notification Contents

Test steps:
1. Update to default config
2. Login as sysadmin and visit Notifications admin console page
- Verify that setting is visible and matches text content
- Verify that the help text is visible and matches text content
- Verify that the option/dropdown is visible and has default value
3. Select each option and save
- Verify that the config is correctly saved in the server

#### Ticket Link
Jira ticket: [MM-16232](https://mattermost.atlassian.net/browse/MM-16232)

Related PR - https://github.com/mattermost/mattermost-webapp/pull/4210
